rpmbuild/copr-rpmbuild.spec

```diff
@@ -114,21 +118,25 @@ Requires: libsemanage-python
 Requires: nosync
 %ifarch x86_64
 # multilib counterpart to avoid: config_opts['nosync_force'] = True
+%if !0%{?openEuler}
```

Comment:
The nosync doesn't work nowadays, https://bugzilla.redhat.com/show_bug.cgi?id=2019329 - and it is perhaps a good time to remove it entirely from our deplist.

Reply:
> The nosync doesn't work nowadays, https://bugzilla.redhat.com/show_bug.cgi?id=2019329 - and it is perhaps a good time to remove it entirely from our deplist.

In our case, we are still using `nosync` in our builder but only with `x86_64&aarch64`, so i will keep it for openEuler :)